src/pages/settings/Profile/Contacts/ContactMethodDetailsPage.js

```diff
@@ -209,7 +209,9 @@ class ContactMethodDetailsPage extends Component {
      * @param {Boolean} isOpen
      */
     toggleDeleteModal(isOpen) {
-        this.setState({isDeleteModalOpen: isOpen});
+        InteractionManager.runAfterInteractions(() => {
+            this.setState({isDeleteModalOpen: isOpen});
+        });
```

Comment:
This caused a slight bug regarding delay in opening of modal. More detail in https://github.com/Expensify/App/issues/33410